trace/ui_redesign_test.py

```diff
@@ -310,3 +287,75 @@ def __init__(self, parent):
         self.setText("|")
         self.setFont(self.breaker_font)
         self.setAlignment(Qt.AlignBottom)
+
+
+class ControlPanel(QWidget):
+    def __init__(self):
+        super().__init__()
+        self.setLayout(QVBoxLayout())
+
+        # Create pv plotter layout
+        pv_plotter_layout = QHBoxLayout()
+        self.layout().addLayout(pv_plotter_layout)
+        self.pv_line_edit = QLineEdit()
+        self.pv_line_edit.setPlaceholderText("Enter PV")
+        self.pv_line_edit.returnPressed.connect(self.add_curve_from_line_edit)
+        pv_plotter_layout.addWidget(self.pv_line_edit)
+        pv_plot_button = QPushButton("Plot")
+        pv_plot_button.clicked.connect(self.add_curve_from_line_edit)
+        pv_plotter_layout.addWidget(pv_plot_button)
+
+        # Create axis & curve view
+        self.axis_list = QVBoxLayout()
+        self.axis_list.addStretch()
+        self.layout().addLayout(self.axis_list)
+        new_axis_button = QPushButton("New Axis")
+        new_axis_button.clicked.connect(self.add_axis)
+        self.layout().addWidget(new_axis_button)
+
+    def add_curve_from_line_edit(self):
+        pv = self.pv_line_edit.text()
+        self.add_curve(pv)
+        self.pv_line_edit.clear()
+
+    @property
+    def plot(self):
+        if not self._plot:
+            parent = self.parent()
+            while not hasattr(parent, "plot"):
+                parent = parent.parent()
+            self._plot = parent.plot
+        return self._plot
+
+    @plot.setter
+    def plot(self, plot):
+        self._plot = plot
+
+    def add_axis(self, name: str = ""):
+        logger.debug("Adding new empty axis to the plot")
+        if not name:
+            counter = len(self.plot.plotItem.axes) - 2
+            while (name := f"Y-Axis {counter}") in self.plot.plotItem.axes:
+                counter += 1
+
+        self.plot.addAxis(plot_data_item=None, name=name, orientation="left", label=name)
+        new_axis = self.plot._axes[-1]
+        #new_axis.setLabel(name, color="black")
```

Comment:
```suggestion
        new_axis.setLabel(name, color="black")
```

I'm partial to setting the label to be black instead of matching the curve's color. This might just be me though ¯\\\_(ツ)\_/¯

Reply:
I turned this off to see what happened and didn't notice the axis color change till later.  I'll update